### PR TITLE
a tool to build gamit station.info files.

### PR DESCRIPTION
**Pull request by ozym**
Simply build an info file based on the sessions listings for each mark, merge together any sessions that are only changing by the configured satellite systems.

### CODE REVIEW

File: tools/siteinfo/firmware.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"time"
+)
+
```
Comment (elidana) [inline tools/siteinfo/firmware.go:6]: please add
```
// firmware conversion numbers for Gamit are listed as comments in Gamit code ~/gg/gamit/makex/settim.f
```

File: tools/siteinfo/firmware.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+var firmwareMap = map[string]string{
+	"unknown":           "0.00",
+	"NP 7.19 / SP 3.04": "7.19",
+	"5.10/3.013":        "5.10",
+	"3.2.32.8":          "3.20",
+	"1.2.5":             "1.20",
```
Comment (elidana) [inline tools/siteinfo/firmware.go:12]: please add
```
        "1E95":              "8.25",
	"1E86":              "8.28",
```

**Review by elidana — APPROVED** [review]
Hi Mark,
for some reason the small modifications done in my pr for the firmware have been lost. 
Nothing major, we can add them later on (or I've added those as comments here).

**Comment by ozym** [comment]
@elidana that wan't the intention, I think things got lost with the rebase. I had done the merges ... but mustn't have got the order right.  Putting in an update now.

**Review by elidana — APPROVED** [review]
Thanks @ozym ! Looks perfect!!
How was the phrase, give it a crack nigel?? :-)

**Comment by ozym** [comment]
A yes ... "worth a crack Nigel" ....

**Pull request merged**